src/renderer/src/routes/pages/Portfolio/index.tsx

```diff
@@ -18,14 +24,43 @@ const PortfolioPage = () => {
           <Separator />
         </div>
         <ul className="w-full max-w-full">
-          <SidebarMenuButton title={t('overview')} to="/portfolio/overview" />
-          <SidebarMenuButton title={t('allActivity')} to="/portfolio/activity" />
-          <SidebarMenuButton title={t('allConnections')} to="/portfolio/connections" />
+          <li>
+            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/overview">
+              {t('overview')}
+            </MenuLink>
+          </li>
+
+          <Separator containerClassName="px-3" />
+
+          <li>
+            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/activity">
+              {t('allActivity')}
+            </MenuLink>
+          </li>
+
+          <Separator containerClassName="px-3" />
+
+          <li>
+            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/connections">
+              {t('allConnections')}
+            </MenuLink>
+          </li>
```

Comment:
Only `<li />` can be into a `<ul />`. Use `border-bottom` or:
```suggestion
          <li>
            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/overview">
              {t('overview')}
            </MenuLink>
            
            <Separator containerClassName="px-3" />
          </li>
          <li>
            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/activity">
              {t('allActivity')}
            </MenuLink>
            
            <Separator containerClassName="px-3" />
          </li>
          <li>
            <MenuLink layoutId="portfolio-menu-link" to="/portfolio/connections">
              {t('allConnections')}
            </MenuLink>
          </li>
```